[EDU-3889] feat: add guide - new template (#762)
aziontech/docs: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/templates-e-integracoes/templates-visao-geral.mdx

</Button>

> #### XXXXXXX

Author: ```suggestion
#### Dynamic and Static File Optimization
```

> O template XXXXXXX contém regras e configurações comuns de cache para melhorar a entrega e desempenho de conteúdo estático e dinâmico, bem como imagens, com base na extensão do arquivo. Além disso, ele também permite o processamento e otimização de imagens. Dessa forma, você pode entregar seu conteúdo usando a Azion como CDN.

Reviewer: ```suggestion
O template XXXXXXX contém políticas e configurações padrão de cache para melhorar a entrega e desempenho de conteúdo estático e dinâmico, bem como imagens, com base na extensão do arquivo. Além disso, ele também permite o processamento e otimização de imagens. Dessa forma, você pode entregar seu conteúdo usando a Azion como CDN.
```

> O template XXXXXXX contém políticas e configurações padrão de cache para melhorar a entrega e desempenho de conteúdo estático e dinâmico, bem como imagens, com base na extensão do arquivo. Além disso, ele também permite o processamento e otimização de imagens. Dessa forma, você pode entregar seu conteúdo usando a Azion como CDN.

Author: ```suggestion
O template **Dynamic and Static File Optimization** contém políticas e configurações padrão de cache para melhorar a entrega e desempenho de conteúdo estático e dinâmico, bem como imagens, com base na extensão do arquivo. Além disso, ele também permite o processamento e otimização de imagens. Dessa forma, você pode entregar seu conteúdo usando a Azion como CDN.
```

> <Button href="/pt-br/documentacao/produtos/guias/XXXXXXXX/" text="consulte o guia do template XXXXXX" variant="secondary"> 

Author: ```suggestion
<Button href="/pt-br/documentacao/produtos/guias/dynamic-static-file-optimization-template/" text="consulte o guia do template Dynamic and Static File Optimization" variant="secondary"> 
```

</Button>
<br />

> <Button href="#" text="implante o template" variant="secondary"> 

Author: ```suggestion
<Button href="https://manager.azion.com/build-application/build/choose-template/template/azion/dynamic-and-static-file-optimization" text="deploy this template" text="implante o template" variant="secondary"> 
```

> <Button href="https://manager.azion.com/build-application/build/choose-template/template/azion/dynamic-and-static-file-optimization" text="deploy this template" text="implante o template" variant="secondary"> 

Author: ```suggestion
<Button href="https://manager.azion.com/build-application/build/choose-template/template/azion/dynamic-and-static-file-optimization" text="implante o template" variant="secondary"> 
```
